JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -0,0 +1,579 @@
+                                                   jaspResults,
+                                                   ready, 
+                                                   positionInContainer){
+
+  if(!options[["summaryTable"]])
+    return()
+
+  .updateTabNumber(jaspResults)
+
+  if(is.null(benfordsLawContainer[["benfordsLawTable"]])){
+
+    tableTitle <- paste0("<b>Table ", 
+                  jaspResults[["tabNumber"]]$object, 
+                  ".</b> Descriptive Statistics")
+
+    benfordsLawTable <- createJaspTable(tableTitle)
+    benfordsLawTable$position <- positionInContainer
+
+    benfordsLawTable$dependOn(options = "summaryTable")
+
+    benfordsLawTable$addColumnInfo(name = 'digit', 
+                                  title = 'Leading digit', 
+                                  type = 'integer')
+    benfordsLawTable$addColumnInfo(name = 'count', 
+                                  title = 'Count', 
+                                  type = 'integer')
+    benfordsLawTable$addColumnInfo(name = 'percentage',  
+                                  title = 'Percentage', 
+                                  type = 'string')
+    benfordsLawTable$addColumnInfo(name = 'inBenford', 
+                                  title = "Benford's law", 
+                                  type = 'string')
+
+    benfordsLawContainer[["benfordsLawTable"]] <- benfordsLawTable
+
+    if(options[["digits"]] == "first"){
+      digits <- 1:9
+    } else {
+      digits <- 10:99
+    }
+
+    if(!ready){
+
+      row <- data.frame(digit = digits, 
+                        count = rep(".", length(digits)),
+                        percentage = rep(".", length(digits)),
+                        inBenford = paste0(round(log10(1 + 1 / digits) * 100, 2), 
+                                            "%"))
+      benfordsLawTable$addRows(row)
+      return()
+    } 
+
+    state <- .auditClassicalBenfordsLawState(dataset, 
+                                            options, 
+                                            benfordsLawContainer,
+                                            ready)
+
+    percentagesLabel <- paste0(round(state[["percentages"]] * 100, 2), "%")
+    inBenfordLabel <- paste0(round(state[["inBenford"]] * 100, 2), "%")
+
+    row <- data.frame(digit = state[["digits"]], 
+                      count = state[["counts"]], 
+                      percentage = percentagesLabel, 
+                      inBenford = inBenfordLabel)
+
+    benfordsLawTable$addRows(row)
+  }
+}
+
+.benfordsLawPlot <- function(dataset, 
+                             options, 
+                             benfordsLawContainer, 
+                             jaspResults,
+                             ready, 
+                             positionInContainer){
+
+  if(!options[["benfordsLawPlot"]])
+    return()
+
+  .updateFigNumber(jaspResults)
+
+  if(is.null(benfordsLawContainer[["benfordsLawPlot"]])){
```

Comment:
you can also 
```suggestion
if (!is.null(benfordsLawContainer[["benfordsLawPlot"]])) return()
```
and then just continue the program